spring-integration-file/src/main/java/org/springframework/integration/file/tail/OSDelegatingFileTailingMessageProducer.java

```diff
@@ -15,17 +15,17 @@
  */
 package org.springframework.integration.file.tail;
 
-import java.io.BufferedReader;
-import java.io.IOException;
-import java.io.InputStreamReader;
-import java.util.Date;
-
 import org.springframework.messaging.MessagingException;
 import org.springframework.scheduling.SchedulingAwareRunnable;
 import org.springframework.scheduling.TaskScheduler;
 import org.springframework.scheduling.concurrent.ThreadPoolTaskScheduler;
 import org.springframework.util.Assert;
 
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Date;
```

Comment:
Minor, but we prefer this imports style: 
![imports idea](https://cloud.githubusercontent.com/assets/1071590/6780155/4e906c82-d16b-11e4-9144-1674e6e4116b.jpg)

Of course, you don't need to follow with this rule, but you should be sure that `import` block hasn't been reformatted before pushing.